hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -5,14 +5,243 @@ title: "Life Cycle Controller Technical Specification"
+
+The isolation ensures three things:
+- The TAP controller is unable to issue instructions that would put the design into scan mode.
+This ensures that secrets cannot be scanned out, and specific values cannot be scanned into the design to emulate a particular functional mode
+- The TAP controller is unable to issue any kind of self test that would disrupt and scramble live logic which could lead to unpredictable behavior
+- The TAP controller or test function is unable to alter the non-volatile contents of flash or OTP
+
+See [TAP isolation]({{< relref "#tap-isolation" >}}) for more implementation details.
+
+### NVM_DEBUG_EN
+
+NVM modules like Flash implement debug access that bypasses memory protection or lockdown.
+This feature may be there for a variety of reasons, but primarily it can be used to debug the normal behavior of the controller.
+
+This type of functionality, if it exists, must be disabled during specific life cycle states.
+Since these backdoor functions may bypass memory protection, they could be used to read out provisioned secrets that are not meant to be visible to software or a debug host.
+
+### HW_DEBUG_EN
+
+Hardware debug enable refers to the general ungating of both invasive (JTAG control of the processor, bidirectional analog test points) and non-invasive debug (debug bus observation, and register access error returns).
+
+This signal thus needs to be routed to all security-aware and debug capable peripherals.
+This signal is used to determine whether OpenTitan peripheral register interfaces should [silently error]({{< relref "doc/rm/register_tool/_index.md#error_responses" >}}).
+If HW_DEBUG_EN is set to ON, normal errors should be returned.
+If HW_DEBUG_EN is set to OFF, errors should return silently.
+
+Similar to DFT_EN, HW_DEBUG_EN is also used to isolate the processor TAP.
+When HW_DEBUG_EN is OFF, the TAP should not be able to perform its' normal debug access, thus preventing an external entity from hijacking the processor.
+
+### CPU_EN
+
+CPU enable controls whether code execution is allowed.
+This is implemented as part of the processor's reset controls.
+In OpenTitan's [reset topology]({{< relref "hw/ip/rstmgr/doc" >}}), it is not possible to reset only the processor by itself, so this reset control extends to a large population of the OpenTitan peripherals.
+
+This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it is not possible for the processor to execute code that breaks the device out of a non-functional state.
+
+In conjunction with DFT_EN / HW_DEBUG_EN, this acts as the final layer in life cycle defense in depth.
+
+### PROVISION_EN
+
+Provision-enable is a separate signal that controls whether the non-volatile provisioning of life cycle related collateral can be accessed (see [OTP]({{< relref "#otp-collateral" >}}) and [flash]({{< relref "#flash-collateral" >}}) sections for list of collateral and also what collateral in each area is affected by provision-enable).
```

Comment:
I think all the other enable does not have a dash in between. Maybe we can unify it if this is not an exception?

Reply:
oh yeah, this should be `PROVISION_EN`